include/mbedtls/ssl_internal.h

```diff
@@ -270,7 +270,15 @@ struct mbedtls_ssl_handshake_params
 #endif
 #if defined(MBEDTLS_ECDH_C)
     mbedtls_ecdh_context ecdh_ctx;              /*!<  ECDH key exchange       */
-#endif
+
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+    psa_ecc_curve_t ecdh_psa_curve;
+    psa_key_slot_t ecdh_psa_privkey;
+    unsigned char ecdh_psa_peerkey[256];
```

Comment:
Should this also be MBEDTLS_PSA_MAX_EC_PUBKEY_LENGTH ?

Reply:
Yes